common/primitives/src/capacity.rs

```diff
@@ -13,6 +14,57 @@ pub trait TargetValidator {
 	fn validate(target: MessageSourceId) -> bool;
 }
 
+#[derive(
+	Clone,
+	Copy,
+	Debug,
+	Decode,
+	Encode,
+	TypeInfo,
+	Eq,
+	MaxEncodedLen,
+	PartialEq,
+	PartialOrd,
+	DecodeWithMemTracking,
+)]
+/// The type of staking a given Staking Account is doing.
+pub enum StakingType {
+	/// Staking account targets Providers for capacity only, no token reward
+	MaximumCapacity,
+	/// New reward program with new rules.
+	/// Defines a mechanism for locking tokens that is both time-sensitive and event-driven, with immutable thawing behavior.
+	CommittedBoost,
```

Comment:
@shannonwells @claireclark1 -- this was renamed from `ProviderBoost`. Just a heads-up in case this ripples to the Javascript side of things; wasn't sure if it could break the Provider Boosting UI.